narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -253,8 +253,8 @@ function get_hotfix_zip_list_file {
 
 			for zip_directory_name in $(lc_curl "${zip_directory_url}/" - | grep -E -o "20[0-9]+\.q[0-9]\.[0-9]+" | uniq)
 			do
-				lc_curl "${zip_directory_url}/${zip_directory_name}/" - | grep -E -o "liferay-dxp-20[a-z0-9\.]+-hotfix-[0-9]{0,9}.zip" | uniq | sed "s@^@${zip_directory_name}/@"
-			done > "${zip_list_file}"
+				lc_curl "${zip_directory_url}/${zip_directory_name}/" - | grep -E -o "liferay-dxp-20[a-z0-9\.]+-hotfix-[0-9]{0,9}.zip" | uniq - "${zip_list_file}"| sed "s@^@${zip_directory_name}/@"
```

Comment:
This should not work.
To my understanding, this creates a file, and sed modifies the `uniq` command output which is not the file's content (probably empty output). Did you test it?

Reply:
I did test it but it didn't show me any errors. Would the -i flag for sed fix the issue?

Reply:
Yes, but not in the pipeline.
It won't throw an error, still won't be appropriate, though.